setup.py

```diff
@@ -25,7 +25,7 @@
 
     packages=[
         'cloudify_gcp',
-        'cloudify_gcp.compute',
```

Comment:
This should not be removed, you're breaking all of the compute types if you do. Just add a new line for `container_engine`

Reply:
@funkyHat this could be removed by mistake, as I said before I'm not done with this PR :)
and thanks for your comments

Reply:
The same for this , I will fix it